client/scss/components/_standfirst.scss

```diff
@@ -0,0 +1,15 @@
+//* @define standfirst; weak
+
+.standfirst {
+  @include font('HNL2');
+  border-left: 5px solid color('black');
+  padding-left: 1em;
+
+  p {
```

Comment:
I would argue that a standfirst should be able to contain more than that - what about lists for example?